Code/GraphMol/MolDraw2D/MolDraw2DQt.h

```diff
@@ -26,26 +26,28 @@ namespace RDKit {
 class RDKIT_MOLDRAW2D_EXPORT MolDraw2DQt : public MolDraw2D {
  public:
   MolDraw2DQt(int width, int height, QPainter &qp, int panelWidth = -1,
-              int panelHeight = -1);
+              int panelHeight = -1, bool noFreetype = false);
 
-  // set font size in molecule coordinate units. That's probably Angstrom for
-  // RDKit. It will turned into drawing units using scale_, which might be
-  // changed as a result, to make sure things still appear in the window.
-  void setFontSize(double new_size);
   void setColour(const DrawColour &col);
 
   void drawLine(const Point2D &cds1, const Point2D &cds2);
   void drawChar(char c, const Point2D &cds);
   void drawPolygon(const std::vector<Point2D> &cds);
   void clearDrawing();
-
+#if 0
```

Comment:
bc these aren't implemented yet?

Reply:
should have removed those, they were vestigial